diff --git a/node_modules/lamejs/lame.all.js b/node_modules/lamejs/lame.all.js
index bfd3637..b905508 100644
--- a/node_modules/lamejs/lame.all.js
+++ b/node_modules/lamejs/lame.all.js
@@ -1,4 +1,3 @@
-function lamejs() {
 function new_byte(count) {
     return new Int8Array(count);
 }
@@ -15511,8 +15510,9 @@ WavHeader.readHeader = function (dataView) {
 
 L3Side.SFBMAX = (Encoder.SBMAX_s * 3);
 //testFullLength();
+export var lamejs = {}
 lamejs.Mp3Encoder = Mp3Encoder;
 lamejs.WavHeader = WavHeader;
-}
+
 //fs=require('fs');
-lamejs();
+//lamejs();
